docker/nginx.Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM nginx:stable-alpine AS base
+
+LABEL maintainer="andy.driver@digital.justice.gov.uk"
+
+FROM controlpanel:latest AS cpanel
```

Comment:
This probably isn't the best way to get the static files into the Docker image.

Reply:
`:latest` is the :fish:y bit